index.html

```diff
@@ -332,6 +332,33 @@ <h3 class="lets-connect">LET’S CONNECT</h3>
       </div>
 
     </section>
+    <section class="contact">
+      <div class="contact-wrap">
+        <div class="contact-intro">
+          <h1 class="contact-title">Contact me</h1>
```

Comment:
- [ ] Please use `h2` tag instead of `h1` for your heading to follow the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) :blush: